src/components/VideoChat/VideoChat.tsx

```diff
@@ -341,7 +341,12 @@ export class VideoChat extends React.Component<VideoChatProps> {
                       }}
                     >
                       <div
-                        title={nameMap[p.id] || p.id}
+                        title={
+                          nameMap[p.id]
+                            ? nameMap[p.id] +
+                              p.id.slice(p.id.lastIndexOf('#')).substring(0, 4)
```

Comment:
maybe we can only show this if there is a user with a duplicate name? Think it's confusing if we always show it.